src/deepcell/data_splitter.py

```diff
@@ -170,7 +211,11 @@ def _sample_dataset(self, dataset: List[ModelInput],
                 self._center_roi_centroid is True or \
                 self._center_roi_centroid == 'test'
 
-        artifacts = [dataset[i] for i in index]
+        selected_experiments = self._exp_ids[index]
+        artifacts = []
```

Comment:
Can this logic to pull all `ModelInput` that belong to `seleceted_experiments` happen outside this method? Because of the decision to not split by experiment for train/val split, this bit of code will be confusing in that case.

Reply:
So convert the experiment train/test indexes into ROI train/test indexes and input them into an unmodified `_sample_dataset` method?

Reply:
I think so. Just construct `dataset` within `get_train_test_split` prior to passing it to this method, so that `_sample_dataset` can stay unmodified.